localstack/utils/aws/aws_stack.py

```diff
@@ -822,7 +822,8 @@ def create_kinesis_stream(stream_name, shards=1, env=None, delete=False):
     if delete:
         run_safe(lambda: stream.destroy(), print_error=False)
     stream.create()
-    stream.wait_for()
+    # Disable this wait_for to reduce create_table operation time
```

Comment:
nit: Instead of commenting out the code, can we remove line 826 and add a short note like this:
```
Note: Returning the stream here without awaiting its creation (via wait_for()), to avoid API call timeouts/retries
```